src/features/payment/components/PaymentChoice/PaymentChoice.tsx

```diff
@@ -1,110 +1,104 @@
-/* eslint-disable sonarjs/cognitive-complexity */
-/* eslint-disable @typescript-eslint/no-unused-expressions */
-/* eslint-disable functional/immutable-data */
 import React from "react";
+import mixpanel from "mixpanel-browser";
+import { PAYMENT_METHODS_CHOICE } from "utils/config/mixpanelDefs";
```

Comment:
FIX with a relative import like the other ones 
`../../../../utils/config/mixpanelDefs`

Reply:
Sorry, I'have amended the last commit fixing the import